components/menu/MenuData.tsx

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {ForwardedRef} from 'react';
+import styled, {css} from 'styled-components';
+import classNames from 'classnames';
+
+import {MenuDataProps} from './Menu.types';
+import MenuPopover from './MenuPopover';
+
+const Divider = () => (
+    <div style={{height: '1px', width: 'auto', backgroundColor: '#e0e0e0'}}/>
+);
+
+const MenuItems = styled.div<{isMobile?: boolean; isSubmenu: boolean}>(
+    ({isMobile, isSubmenu}) => {
+        return isMobile ?
```

Comment:
could we consolidate shared styles and only append some for the different use-cases?